influxdb/test/cases/test-auto-setup-init-bind

```diff
@@ -18,9 +18,9 @@ declare -ra docker_run_influxd=(
     -e DOCKER_INFLUXDB_INIT_PASSWORD=${TEST_PASSWORD}
     -e DOCKER_INFLUXDB_INIT_ORG=${TEST_ORG}
     -e DOCKER_INFLUXDB_INIT_BUCKET=${TEST_BUCKET}
-    -e INFLUXD_HTTP_BIND_ADDRESS=:3333
+    -e INFLUXD_HTTP_BIND_ADDRESS=:2222
     -e INFLUXD_INIT_PORT=9998
-    influxdb:${tag} influxd run
+    influxdb:${tag} influxd run --http-bind-address ":3333"
```

Comment:
I'm probably missing something obvious here, but doesn't specifying this command have the same effect as setting the `INFLUXD_HTTP_BIND_ADDRESS` on line 21?

Reply:
This one should "win" over the env var. I've included both because it's possible for the `entrypoint.sh` to fail to override each one in different ways.

I'll add a comment explaining.